usr/bin/nxdk-c++

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
What about platforms without bash?

Reply:
I don't seem to find any explicit bash features in use? One could discuss to switch to bash as it available on more system, afaik?

Reply:
I'm mostly concerned with Windows (also with running Windows nxdk via CMake instead of WSL or MSYS2).

Python could be a suitable option, because we might even end up having some programs written in Python in their entirety. There are also many options to compile / pack Python to EXE.